config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -49,7 +49,7 @@ presubmits:
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
       fork-per-release: "true"
-
```

Comment:
nit: inconsistent newline removal

Reply:
huh, weird, looks like there was two blanks before. I'll put it back?